tests/karma/unit/crmCaseTypeSpec.js

```diff
@@ -292,21 +311,25 @@ describe('crmCaseType', function() {
   });
 
   describe('CaseTypeListCtrl', function() {
-    var caseTypes, crmApiSpy;
+    var caseTypes, caseTypeCategoriesIndex, crmApiSpy;
 
     beforeEach(function() {
+      var caseTypeCategories = getCaseTypeCategoriesSampleData();
+
       caseTypes = {
         values: {
           1: { id: 1 },
           2: { id: 2 },
           3: { id: 3 }
         }
       };
+      caseTypeCategoriesIndex = _.indexBy(caseTypeCategories.values, 'value');
```

Comment:
Would such a variable name be more self explanatory? `expectedCaseTypeCategoriesIndexed`